fix: clarify that a plugin filename is needed (#6083)
influxdata/docs-v2: content/shared/influxdb3-cli/create/trigger.md

## Options

| Option |                     | Description                                                                                              |
| :----- | :------------------ | :------------------------------------------------------------------------------------------------------- |
| `-H`   | `--host`            | Host URL of the running {{< product-name >}} server (default is `http://127.0.0.1:8181`)                 |
| `-d`   | `--database`        | _({{< req >}})_ Name of the database to operate on                                                       |
|        | `--token`           | _({{< req >}})_ Authentication token                                                                     |

> |        | `--plugin-filename` | _({{< req >}})_ Python file name of the file on the server's plugin-dir containing the plugin code.      | 

Reviewer: ```suggestion
|        | `--plugin-filename` | _({{< req >}})_ Name of the file, stored in the server's `plugin-dir`, that contains the Python plugin code to run     | 
```
